examples/thread_duel/README.md

```diff
@@ -0,0 +1,9 @@
+Thread-Duel
+== == == == == ==
+
+this is a thread duel application
+    to show or not to show
+RIOTS multi-threading abilities
```

Comment:
Can you explain more here? Right now when looking at the test and example I'm not sure what to expect...